hbw/app/javascript/packs/hbw/widget.js.jsx

```diff
@@ -44,7 +44,8 @@ modulejs.define(
           entity_code:      this.options.entity_code,
           initialVariables: payload.variables,
           fetch_all:        this.options.fetch_all,
-          poll_interval:    this.options.poll_interval
+          poll_interval:    this.options.poll_interval,
+          taskListPath:     this.options.taskListPath,
```

Comment:
Why do we mix camelCase with snake_case?

Reply:
Bad thing, will fix it later.